firebase-vscode/webviews/EmulatorPanel.tsx

```diff
@@ -74,7 +80,9 @@ export function EmulatorPanel({
   });
 
   broker.on("notifyEmulatorImportFolder", ({ folder }) => {
-    webLogger.debug(`notifyEmulatorImportFolder received in sidebar: ${folder}`);
+    webLogger.debug(
+      `notifyEmulatorImportFolder received in sidebar: ${folder}`
+    );
     emulatorUiSelections.importStateFolderPath = folder;
```

Comment:
Sorry I didn't notice this before but if you're using React state you probably shouldn't be mutating the state variable directly? Same in other places.

Reply:
In that case - I updated this to create a new object and set that instead. I didn't find any problems with the former but I can see how it might make things act strangely.